### ISSUE TITLE
How to remove prefix "about ...."?

### ISSUE THREAD

**Issue by ahmetsulek** [egoist/vue-timeago#94]
Hey everyone,

I really didn't understand how to remove the prefix. 

I would like to see" _**5 hours ago**_" instead of "_**about 5 hours ago**_"

Can you please provide a code example?

Best,

**Comment by vesper8** [comment]
I would also like to do this but I don't think it's currently possible

I got as far as using my own custom converter. But in order to do the above you would have to create your own custom locale and replace the word 'about' by whatever you wanted.

A PR to date-fns adding a locale factory would probably make this easier

https://github.com/date-fns/date-fns/issues/1047

**Comment by egoist** [comment]
I think you can use https://date-fns.org/v1.30.1/docs/distanceInWordsStrict which removes helpers like 'almost', 'over', 'less than' ?

**Comment by malinushj** [comment]
@egoist thanks for the solution!!
tldr; here's how I made it work:
```js
Vue.use(VueTimeago, {
  locale: 'en',
  converter(date, locale, { includeSeconds, addSuffix = true }) {
    const distanceInWordsStrict = require('date-fns/distance_in_words_strict')
    return distanceInWordsStrict(Date.now(), date, { locale, addSuffix, includeSeconds });
  }
});
```

**Comment by mauxtin** [comment]
Just in case someone finds this helpful, I did the following instead of @malinushj's solution because that one overrides the defaults. In my case I only wanted to remove the word "about".

```
  converter(date, locale, converterOptions) {
    const { includeSeconds, addSuffix = true } = converterOptions
    return toNow(date, {
      locale,
      includeSeconds,
      addSuffix
    }).replace('about ', '')
  }
```

**Comment by inspire22** [comment]
@mauxtin where's "toNow" come from?

**Comment by jesusthatsgreat** [comment]
> @mauxtin where's "toNow" come from?

above this, there should be an import statement:

`import toNow from 'date-fns/distance_in_words_to_now'`

Otherwise you'll get an error because toNow doesn't exist.

**Comment by uwejan** [comment]
`import toNow from 'date-fns/distance_in_words_to_now'`
@jesusthatsgreat can not find this import, i tried 
`import formatDistanceToNow from 'date-fns/formatDistanceToNow/index'`
i get this error;
`[Vue warn]: Error in data(): "RangeError: Invalid time value"`

**Comment by viktoriiakrause** [comment]
@uwejan you can look for date-fns inside vue-timeago package. I believe there is locked version of date-fns.
In vue-timeago version 5.1.3, for example, I can import distance_in_words_strict

```
import distanceInWordsStrict from 'vue-timeago/node_modules/date-fns/distance_in_words_strict'
```

But in this case you may need to lock version for vue-timeago(a precaution of updating date-fns inside vue-timeago with following code import failure)

There is also a native solution, looks really great (haven't tested it yet though). 
```
const relativeFormat = new Intl.RelativeTimeFormat('en', { style: 'narrow' });

console.log(relativeFormat.format(-2, 'hour')); // 2 hr. ago
console.log(relativeFormat.format(1, 'day')); // in 1 day
```
